README.md

```diff
@@ -12,9 +12,17 @@ https://libyear.com/
 
 ![Demo Image](./docs/demo.png)
 
-## How to install
-`pip install libyear`
+## Description
 
+This project is used for python project (pip and poetry) to measure what is your libyear dependencies.
```

Comment:
For poetry you need to generate a `requirements.txt` file with a command similar to `poetry export -f requirements.txt --output requirements.txt`.

Also, we I think "measure what is your libyear dependencies" it maybe not that clear. "calculate the libyear metric for your dependencies" might be bit easier to understand.